application/config/database.php

```diff
@@ -47,20 +47,20 @@
 | The $active_group variable lets you choose which connection group to
 | make active.  By default there is only one group (the 'default' group).
 |
-| The $active_record variables lets you determine whether or not to load
+| The $query_builder variables lets you determine whether or not to load
 | the active record class
 */
 
 $active_group = 'default';
-$active_record = TRUE;
+$query_builder = TRUE;
 
 $db['default']['hostname'] = getenv('DB_HOST') ?: '';
 $db['default']['username'] = getenv('DB_USER') ?: '';
-$db['default']['password'] = getenv('DB_PASSWORD' ?: '');
+$db['default']['password'] = getenv('DB_PASSWORD') ?: '';
```

Comment:
I guess this would always evaluate to `$db['default']['password'] = getenv('DB_PASSWORD')` so as long as you had DB_PASSWORD env set, it would work? Only if the variable is unset, would it crash?

Reply:
Oh right, duh.